src/bika/lims/content/bikasetup.py

```diff
@@ -200,6 +200,21 @@ def getCounterTypes(self, instance=None):
                           "this option will be checked and readonly.")
         )
     ),
+    # NOTE: This is a Proxy Field which delegates to the SENAITE Registry!
+    BooleanField(
+        "EnableGlobalAuditlog",
+        schemata="Security",
+        default=False,
+        widget=BooleanWidget(
+            label=_("Enable global Auditlog"),
```

Comment:
should we be consistent with https://github.com/senaite/senaite.core/blob/c363f53fe15846320d31eb3f4f24fec5631e07b7/src/bika/lims/profiles/default/types/AuditLog.xml#L8 here?

```suggestion
            label=_("Enable global Audit Log"),
```

Reply:
Thanks for pointing this out @winniehell!